src/ui/src/partials/system_index.html

```diff
@@ -1,44 +1,39 @@
-<h1 class='page-header'>Available Systems</h1>
 
 <fetch-data response="response"></fetch-data>
 
-<div id="systems-container"
-     class="animate-if"
-     ng-if="responseState(response) === 'success'">
-
-  <div ng-repeat="system in data" class="container-item panel panel-default">
-    <div class="panel-heading">
-      <h3>
-        <span ng-class="util.getIcon(system.icon_name)"></span>
-        <span>{{system.display_name || system.name}} v{{system.version}}</span>
-      </h3>
-    </div>
-    <div class="panel-body" style="padding-bottom: 0;">
-      <div class="container-fluid">
-        <div class="row">
-          <div class="col-sm-4 system-table table-header">Description</div>
-          <div class="col-sm-8 system-table">{{ system.description }}</div>
-        </div>
-        <div class="row">
-          <div class="col-sm-4 system-table table-header">Commands</div>
-          <div class="col-sm-8 system-table">{{ system.commands.length }}</div>
-        </div>
-        <div class="row">
-          <div class="col-sm-4 system-table table-header">Instances</div>
-          <div class="col-sm-8 system-table">{{ system.instances.length }}</div>
-        </div>
-      </div>
-    </div>
-    <div class="panel-body" style="padding-top: 0;">
-      <div>
-        <button type="button" class="btn btn-primary btn-lg btn-block"
-            ng-click="exploreSystem(system)" style="margin-top: 10px;">
-          Explore {{system.name}} commands
-        </button>
-      </div>
-    </div>
+<div ng-if="responseState(response) === 'success'">
+  <h1 class='page-header' ng-bind-html="getBreadCrumbs()"></h1>
```

Comment:
I think this should be tweaked before merging. It's really constructing HTML in the controller, which we should avoid if we can (I know there are places where we don't follow this rule, but I think we should try to if possible). 
```
ng-bind-html="getBreadCrumbs()"
```

We do something similar in the request view for children requests:

```

  <ol class="tagline breadcrumb" ng-show="request_tree.length > 1">
    <li ng-repeat="request in request_tree = getParentTree(data)" ng-class="{active: $last}"
      ><span ng-if="!$last"><a ui-sref="base.namespace.request({requestId: request.id})">{{request.command}}</a></span
      ><span ng-if="$last">{{request.command}}</span>
    </li>
  </ol>
```

Reply:
Changed this to pulled the variables from the `$scope` object. Also wrote it so if we copy and past it somewhere else, it "should" work, like in on the commands page.